models/project/src/mc-project/applied-action-right/generator.ts

```diff
@@ -0,0 +1,22 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import type { TAppliedActionRight } from './types';
+
+const fakeActionRightNames = [
+  'canAddPrices',
+  'canEditPrices',
+  'canAddProducts',
+  'canPublishProducts',
+  'canCreateReturns',
+  'canSendQuote',
+];
+
+const generator = Generator<TAppliedActionRight>({
+  fields: {
+    // TODO: use more realistic values for action rights
+    group: fake((f) => f.animal.type()),
```

Comment:
Can you maybe also update the group values to have something realistic? 🙏

Reply:
updated [d22c170](https://github.com/commercetools/test-data/pull/464/commits/d22c1709dcdb112fb8b998edf20afcaf2328be63)